apps/etterlatte-behandling/src/main/kotlin/behandling/revurdering/RevurderingRoutes.kt

```diff
@@ -33,7 +33,11 @@ internal fun Route.revurderingRoutes(
             }
 
             generellBehandlingService.hentSenestIverksatteBehandling(sakId)?.let { forrigeIverksatteBehandling ->
-                val revurdering = revurderingService.opprettRevurdering(forrigeIverksatteBehandling, body.aarsak)
+                val revurdering = revurderingService.opprettManuellRevurdering(
+                    sakId = null,
```

Comment:
Skal det vera sakId null her? Ser umiddelbart litt rart ut.

Reply:
Ja - denne metoden kalles fra to steder.

Herfra sender vi ikke sakId. Fra omregning så er sakId med.

Hvis den sendes inn null så brukes forrige behandlings sakId

Reply:
Men - kunne kanskje bare sendt inn forrigeIverksatteBehandling sin sakId her og gjort den non-nullable.

Reply:
Gjort